src/Cli/dotnet/ProjectInstanceExtensions.cs

```diff
@@ -22,7 +21,8 @@ public static string GetDefaultProjectTypeGuid(this ProjectInstance projectInsta
             string projectTypeGuid = projectInstance.GetPropertyValue("DefaultProjectTypeGuid");
             if (string.IsNullOrEmpty(projectTypeGuid) && projectInstance.FullPath.EndsWith(".shproj", StringComparison.OrdinalIgnoreCase))
             {
-                projectTypeGuid = ProjectTypeGuids.SharedProjectGuid;
+                // TODO: Centralize project type guids
```

Comment:
TODO

Reply:
I think maybe this will be refactored, as well as all other places in the sdk where guids are hard-coded, but maybe that should be a different PR :)